Source/WebKit/WebKit.xcodeproj/project.pbxproj

```diff
@@ -7543,6 +7551,7 @@
 		EB450E0E2996C7A1009724B1 /* WKWebsiteDataStoreRefPrivateMac.mm */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.cpp.objcpp; name = WKWebsiteDataStoreRefPrivateMac.mm; path = mac/WKWebsiteDataStoreRefPrivateMac.mm; sourceTree = "<group>"; };
 		EB579C3529AEBCF100894C1C /* EarlyHintsResourceLoader.cpp */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.cpp.cpp; path = EarlyHintsResourceLoader.cpp; sourceTree = "<group>"; };
 		EB579C3629AEBCF100894C1C /* EarlyHintsResourceLoader.h */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.c.h; path = EarlyHintsResourceLoader.h; sourceTree = "<group>"; };
+		EB7169D22B05D8B2005F305D /* DerivedSources/WebKit/WKApplicationUtilities.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = DerivedSources/WebKit/WKApplicationUtilities.swift; sourceTree = BUILT_PRODUCTS_DIR; };
```

Comment:
nit: Derived sources typically go in the "Derived Sources" group, with their location set to "Relative to Group".

Reply:
Yeah this is definitely not the way I meant to include the file into the project. It's kind of surprising it even builds. Will fix.